acoustid/config.py

```
@@ -554,6 +554,18 @@ def read_env(self, prefix):
         pass  # XXX
 
 
+class SentryConfig(BaseConfig):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)